### PR TITLE
add basic test(s)

### PR DESCRIPTION
**Pull request by afeld**
Going to keep going with adding more! If this looks ok, I say we merge it, and I can send separate/smaller PRs as follow up.

### CODE REVIEW

File: templater/templater.go [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+	"fmt"
+
+	// using fork because of https://github.com/moovweb/gokogiri/pull/93#issuecomment-215582446
+	"github.com/jbowtie/gokogiri"
+	"github.com/jbowtie/gokogiri/xml"
+	"github.com/opencontrol/doc-template/docx"
+)
+
+func GetWordDoc(path string) (doc *docx.Docx) {
+	doc = new(docx.Docx)
```
Comment (jcscottiii) [inline templater/templater.go]: check the error

File: templater/templater.go [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+	doc.ReadFile(path)
```
Comment (jcscottiii) [inline templater/templater.go]: return the error from reading the file

File: templater/templater.go [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+	content := roleCell.Content()
+	// TODO remove hard-coding
+	content += " {{getResponsibleRole \"NIST-800-53\" \"AC-2 (1)\"}}"
+	roleCell.SetContent(content)
+
+	return
+}
+
+func templatizeXMLDoc(doc *xml.XmlDocument) (err error) {
+	tables, err := findControlEnhancementTables(doc)
```
Comment (jcscottiii) [inline templater/templater.go:76]: thinking long term, when we start to add more of these entrypoint functions to injecting various tags. should this logic be all wrapped up in a function?

**Comment by jcscottiii** [comment]
just a few things

File: templater/templater.go (hunk 3 above)
Comment (afeld) [inline templater/templater.go:76]: Can you elaborate on what you mean?

Comment (jcscottiii) [inline templater/templater.go:76]: in my head (which could be wrong), i imagined `templatizeXMLDoc` would look like:

``` go
func templatizeXMLDoc(doc *xml.XmlDocument) (err error) {
    err := addControlEnchancementTablesTags(doc)
    if err != nil {
       return nil
    }
    err = addControlSectionTags(doc)
    if err != nil {
       return nil
    }
    // so on
}
```

File: templater/templater.go [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+// modifies the `table`
+func FillTable(table xml.Node) (err error) {
+	roleCell, err := findResponsibleRoleCell(table)
+	if err != nil {
+		return
+	}
+
+	content := roleCell.Content()
+	// TODO remove hard-coding
+	content += " {{getResponsibleRole \"NIST-800-53\" \"AC-2 (1)\"}}"
```
Comment (afeld) [inline templater/templater.go:69]: This (reported in Code Climate) will be fixed by https://github.com/opencontrol/fedramp-templater/pull/8.

**Comment by afeld** [comment]
Updated!

**Comment by jcscottiii** [comment]
Run https://github.com/golang/lint too.

**Comment by afeld** [comment]
Taken care of by https://github.com/opencontrol/fedramp-templater/pull/6.

**Pull request merged**